### PR TITLE
expression in subcription subject, not in conditions

### PR DESCRIPTION
**Pull request by crbrox**
@fgalan 
@kzangeli

### CODE REVIEW

**Comment by kzangeli** [comment]
LGTM, CNR ?

**Comment by fgalan** [comment]
CNR entry not needed (as this is an addition to something previously add as part of the present release)

File: src/lib/ngsi10/SubscribeContextRequest.h [hunk 1]
```diff
@@ -54,6 +54,13 @@ typedef struct SubscribeContextRequest
   Throttling             throttling;             // Optional
   int64_t                expires;
 
+  struct {
```
Comment (fgalan) [inline src/lib/ngsi10/SubscribeContextRequest.h]: The example in data_model.md (check file name, I'm not sure) should be fixed, to put expression in the first level.

Comment (crbrox) [inline src/lib/ngsi10/SubscribeContextRequest.h]: Fixed dcddd15

**Comment by fgalan** [comment]
LGTM

**Pull request merged**